addons/l10n_es_pos/tests/test_frontend.py

```diff
@@ -41,6 +41,58 @@ def get_number_of_regular_invoices():
         self.assertEqual(num_of_simp_invoices, 3)
         self.assertEqual(num_of_regular_invoices, 1)
 
+    def test_l10n_es_pos_reconcile(self):
+        if not self.env["ir.module.module"].search([("name", "=", "pos_settle_due"), ("state", "=", "installed")]):
+            self.skipTest("pos_settle_due module is required for this test")
+
+        # create customer account payment method
+        self.customer_account_payment_method = self.env['pos.payment.method'].create({
+            'name': 'Customer Account',
+            'split_transactions': True,
+        })
+        # add customer account payment method to pos config
+        self.main_pos_config.write({
+            'payment_method_ids': [(4, self.customer_account_payment_method.id, 0)],
+        })
```

Comment:
Please either use Command or just this
`self.main_pos_config.payment_method_ids = self.customer_account_payment_method.id`